daisy_workflows/image_build/windows/windows-server-2022-dc-uefi-byol.wf.json

```diff
@@ -1,106 +0,0 @@
-{
```

Comment:
What were these two deleted images used for?

Reply:
Windows Server 2022 was released on Aug 18, 2021, so it cannot have been purchased before October 1, 2019

Reply:
Windows Server 2022 BYOL, but since no customer can BYOL the images they need to be removed.